integrations/pull_create_test.go

```diff
@@ -59,4 +59,12 @@ func TestPullCreate(t *testing.T) {
 	assert.Regexp(t, "\\+Hello, World \\(Edited\\)", resp.Body)
 	assert.Regexp(t, "^diff", resp.Body)
 	assert.NotRegexp(t, "diff.*diff", resp.Body) // not two diffs, just one
+
+	// check .patch can be accessed and matches performed change
+	req = NewRequest(t, "GET", url+".patch")
+	resp = session.MakeRequest(t, req, http.StatusOK)
+	assert.Regexp(t, "\\+Hello, World \\(Edited\\)", resp.Body)
```

Comment:
Why not <code>\`\\+Hello, World \\(Edited\\)\`</code> (raw string)? Seems cleaner to me